functional/basic-attestation-with-unpriviledged-agent/test.sh

```diff
@@ -50,16 +45,20 @@ rlJournalStart
         rlRun "limeStartRegistrar"
         rlRun "limeWaitForRegistrar"
         # do special configuration for the agent
-        rlRun "limeUpdateConf ${AGENT_CONFIG_SECTION} run_as ${AGENT_USER}:${AGENT_GROUP}"
+        if limeIsPythonAgent; then
+            rlRun "limeUpdateConf agent run_as ${AGENT_USER}:${AGENT_GROUP}"
+        else
+            rlRun "limeUpdateConf agent run_as '\"${AGENT_USER}:${AGENT_GROUP}\"'"
```

Comment:
Is this something that won't work with Python agent?

Reply:
After trying to make them the same, I realized that the python agent does not strip the quotes from the strings obtained from the configuration in all cases (this being one of them)

So no, while the TOML format requires the quotes, the python agent does not work correctly when the strings are quoted.

Reply:
I see.